.github/workflows/docker_tests.yaml

```diff
@@ -0,0 +1,23 @@
+name: Tests in Docker (3.8)
+
+on:
+  pull_request:
```

Comment:
You can rm it since you have it only for `push` event. Or do you want to see a list of all project tests?

Reply:
yeah, I want to see the whole list